CHANGELOG.md

```diff
@@ -10,15 +10,18 @@ To publish a new release run `scripts/release` from the project directory.
 ### Added
 - Release builds now perform full link-time optimization to create a smaller
   binary.  We do recommend to also `strip` the `mdcat` binary.
-- Render SVG images in [kitty] (see [GH-114]).
```

Comment:
Why did you change this?

Reply:
I spent over an hour fighting with git on merge errors and attempting to collapse git messages.  It was a cut and paste error.

Reply:
Then please undo this particular change.